modules/connatixBidAdapter.js

```diff
@@ -140,6 +158,52 @@ export const spec = {
 
     _handleEids(requestPayload, validBidRequests);
 
+    if (window.pbjs) {
+      window.pbjs.onEvent(EVENTS.AUCTION_TIMEOUT, (timeoutData) => {
+        const isConnatixTimeout = timeoutData.bidderRequests.some(bidderRequest => bidderRequest.bidderCode === BIDDER_CODE);
+
+        if (isConnatixTimeout) {
+          const timeout = timeoutData.timeout;
+          // eslint-disable-next-line no-console
+          console.log(timeout);
```

Comment:
do we need these console.log occurences ?

Reply:
yes, for testing